tests/unit/phpunit/modules/AOR_Reports/AOR_ReportTest.php

```diff
@@ -30,31 +32,6 @@ public function testbean_implements()
 
     public function testsave()
     {
-        $state = new SuiteCRM\StateSaver();
-        $state->pushTable('aod_indexevent');
-        $state->pushTable('tracker');
-        $state->pushTable('aod_index');
-        $state->pushTable('aor_charts');
-        $state->pushTable('aor_fields');
-        $state->pushTable('aor_reports');
-        $state->pushFile('modules/AOD_Index/Index/Index/read.lock.file');
-        $state->pushFile('modules/AOD_Index/Index/Index/segments_3n');
-        $state->pushFile('modules/AOD_Index/Index/Index/read-lock-processing.lock.file');
-        $state->pushFile('modules/AOD_Index/Index/Index/write.lock.file');
```

Comment:
Worth noting the `pop`/`pushFiles` that this PR removes so we can go back and add whatever new solution we come up with.

Given that this means the tests now leave behind files, have you tried running the test suite more than once locally to see if it still works?

Reply:
Ran a few dozens times without any issues (Although I imagine there might be some I just got lucky enough to miss).